Those 2 lines are incorrect, a #define cannot contain a #pragma.
See: https://gcc.gnu.org/onlinedocs/gcc-8.3.0/cpp/Pragmas.html
This was seemingly ignored by previous versions of GCC but it now breaks compilation.

The instances of "#pragma" could be replaced by "_Pragma()", as suggested under the
URL above, but that actually breaks xenmgr, which uses some of the functions that
would otherwise be hidden.

Doing nothing replicates previous behavior and fixes the issue.

--- a/includes/Rts.h
+++ b/includes/Rts.h
@@ -58,8 +58,8 @@ extern "C" {
 #endif
 
 #if __GNUC__ > 4
-#define BEGIN_RTS_PRIVATE #pragma GCC visibility push(hidden)
-#define END_RTS_PRIVATE   #pragma GCC visibility pop
+#define BEGIN_RTS_PRIVATE
+#define END_RTS_PRIVATE
 #else
 #define BEGIN_RTS_PRIVATE /* disabled: BEGIN_RTS_PRIVATE */
 #define END_RTS_PRIVATE   /* disabled: END_RTS_PRIVATE */
